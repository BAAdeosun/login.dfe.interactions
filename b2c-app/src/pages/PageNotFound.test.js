import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import PageNotFound from './PageNotFound';

it('renders without crashing', () => {
    shallow(<PageNotFound />);
});

it('renders correctly', () => {
    const tree = renderer
        .create(<PageNotFound />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});