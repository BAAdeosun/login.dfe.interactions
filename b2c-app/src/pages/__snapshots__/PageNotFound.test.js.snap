// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  id="pageNotFound"
>
  <div>
    <div
      className="govuk-width-container"
    >
      <div
        className="govuk-breadcrumbs"
      >
        <ol
          className="govuk-breadcrumbs__list"
        >
          <li
            className="govuk-breadcrumbs__list-item"
          >
            <a
              className="govuk-breadcrumbs__link"
              href="https://nationalcareers.service.gov.uk/"
            >
              Home
            </a>
          </li>
        </ol>
      </div>
      <main
        className="govuk-main-wrapper"
      >
        <div
          className="govuk-grid-row"
        >
          <div
            className="govuk-grid-column-two-thirds"
          >
            <h1
              className="govuk-heading-xl govuk-!-margin-top-6 govuk-!-margin-bottom-9"
            >
              Page not found
            </h1>
            <div>
              <p
                className="govuk-body"
              >
                If you typed the web address or used a bookmark, check it is correct.
              </p>
              <p
                className="govuk-body"
              >
                If you pasted the web address, check you copied the whole address.
              </p>
              <p
                className="govuk-body"
              >
                If the web address is correct you can 
                <a
                  className="govuk-link"
                  href={null}
                  onClick={null}
                >
                  contact us
                </a>
                 to get help using this service or call 0800 100 900, 8am to 10pm, 7 days a week.
              </p>
            </div>
          </div>
        </div>
      </main>
    </div>
  </div>
</div>
`;
